dev-libs/date/files/date-3.0.1-remove-failing-tests.patch

```diff
@@ -0,0 +1,37 @@
+From: Filip Kobierski <fkobi@pm.me>
+Date: Mon, 2 Sep 2024 13:21:50 +0200
+Subject: [PATCH] remove failing tests
+
+Here are the failiure messages:
+test_c(): Assertion `!in.fail()' failed.
+test_Ip(): Assertion `tp == sys_days{2016_y/12/11} + hours{13}' failed.
+test_p():  Assertion `tp == sys_days{2016_y/12/11} + hours{23}' failed.
```

Comment:
Localization issue?

Reply:
> I am not sure what you mean but I am using `en_US.utf8` for everything

there is also `timezone` settings, and date format may depend on `timezone`. no idea what those tests expect to be set.
btw., skipped tests waiting to be reviewed / debugged are sometimes the main blockers of [stabilization](https://bugs.gentoo.org/showdependencytree.cgi?id=918647&hide_resolved=1)

Reply:
> `/etc/localtime` is a symlink to `../usr/share/zoneinfo/Europe/Warsaw`. Is there anything else I could check?

what i remember from another package is [TZ](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-java/icu4j/icu4j-75.1.ebuild#n97), again no idea if what TZ your package expects.